server/devdocs/index.js

```diff
@@ -59,18 +59,18 @@ function listDocs( filePaths ) {
 				title: doc.title,
 				snippet: defaultSnippet( doc )
 			};
-		} else {
-			return {
-				path: path,
-				title: 'Not found: ' + path,
-				snippet: ''
-			};
 		}
+
+		return {
+			path: path,
+			title: 'Not found: ' + path,
+			snippet: ''
+		};
 	} );
 	return results;
 }
 
-/**
```

Comment:
Yeah, it is a function but it's not exported so I went with "make it just a comment" here.